app/src/androidTest/java/com/duckduckgo/app/httpsupgrade/api/HttpsBloomFilterSpecJsonTest.kt

```diff
@@ -27,7 +27,7 @@ class HttpsBloomFilterSpecJsonTest {
     fun whenGivenValidJsonThenParsesCorrectly() {
         val moshi = Moshi.Builder().build()
         val jsonAdapter = moshi.adapter(HttpsBloomFilterSpec::class.java)
-        val result = jsonAdapter.fromJson(json())
+        val result = jsonAdapter.fromJson(json())!!
```

Comment:
Will this crash if the json is malformed?

Reply:
Yes, and I believe this is the same behavior as we currently have. In this instance, the test will crash which is no big deal. In production, I think RxJava will catch the problem and it'll be a "sync failed" scenario.